content/tests/styling.qmd

```diff
@@ -0,0 +1,932 @@
+
+{{< include /content/include/_support-button.qmd >}}
+{{< include /content/include/_support-outline-button.qmd >}}
+
+## Alerts
+
+::: {.alert .alert-success}
+This is a success alert.
+:::
+
+```md
+::: {.alert .alert-success}
+This is a success alert.
+:::
+```
+
+::: {.alert .alert-danger}
+This is a danger alert.
+:::
+
+```md
+::: {.alert .alert-danger}
+This is a danger alert.
+:::
+```
+
+## Code Blocks with Syntax Highlighting
+
+### Default
+
+```default
+code
+```
+
+### Python
+```python
+# Comment
+import sys
+import subprocess
+
+sys.path.append(
+    subprocess.check_output(["grass", "--config", "python_path"], text=True).strip()
+)
+
+import grass.script as gs
+import grass.jupyter as gj
+```
+
+### Terminal
+```bash
+grass "~/grassdata/" --exec script.py 
+```
+
+### R
+```r
+library(rgrass)
+
+session <- initGRASS(gisBase = "/usr/lib/grass84", # where grass binaries live, `grass --config path`
+                     gisDbase = "/home/user/grassdata", # path to grass database or folder where your project lives
+                     location = "nc_basic_spm_grass7", # existing project name
+                     mapset = "PERMANENT" # mapset name
+                     )
+
+```
+
+### Raw Content
+
+```{=html}
+<iframe src="https://grass.osgeo.org/" width="500" height="400"></iframe>
+```
+
+```md
+\```{=html}
+<iframe src="https://grass.osgeo.org/" width="500" height="400"></iframe>
+```\
+```
+
+## Diagrams
+
+### Flowchart
+```{mermaid}
+flowchart LR
+  A[Hard edge] --> B(Round edge)
+  B --> C{Decision}
+  C --> D[Result one]
+  C --> E[Result two]
+```
+
+### Sequence Diagram
+```{mermaid}
+sequenceDiagram
+  participant Alice
+  participant Bob
+  Alice->>John: Hello John, how are you?
+  loop Healthcheck
+    John->>John: Fight against hypochondria
+  end
+  Note right of John: Rational thoughts <br/>prevail!
+  John-->>Alice: Great!
+  John->>Bob: How about you?
+  Bob-->>John: Jolly good!
+```
+
+### Dot
+
+```{dot}
+graph G {
+  layout=neato
+  run -- intr;
+  intr -- runbl;
+  runbl -- run;
+  run -- kernel;
+  kernel -- zombie;
+  kernel -- sleep;
+  kernel -- runmem;
+  sleep -- swap;
+  swap -- runswap;
+  runswap -- new;
+  runswap -- runmem;
+  new -- runmem;
+  sleep -- runmem;
+}
+```
+
+## Videos
+
+Use the [video shortcode](https://quarto.org/docs/authoring/videos.html)
+
+{{< video https://www.youtube.com/embed/zboP3Z7VBuU >}}
+
+## Page Break
+
+page 1
+
+{{< pagebreak >}}
+
+page 2
+
+## Divs & Spans
+
+### Boarder
+
+::: {.border}
+This content can be styled with a border
+:::
+
+```md
+::: {.border}
+This content can be styled with a border
+:::
+```
+
+### Sidebar
+
+::::: {#special .sidebar}
+
+::: {.warning}
+Here is a warning.
+:::
+
+More content.
+:::::
+
+```md
+::::: {#special .sidebar}
+
+::: {.warning}
+Here is a warning.
+:::
+
+More content.
+:::::
+```
+
+### Spans
+
+[This is *some text*]{.class key="val"}
+
+<span class="custom-span">This is a custom span.</span>
```

Comment:
These 2 do not render anything special... is that right?

Reply:
I will double check.

Reply:
I've updated this.